app/src/main/java/edu/uc/reedws/musiclink/service/PlaylistService.kt

```diff
@@ -18,12 +18,20 @@ class PlaylistService(application: Application) {
         getLocalPlaylistDAO()
     }
 
+    /**
+     * Fetch the playlists added by user
+     *
+     */
```

Comment:
Added appropriate comments. Since this function is calling get playlist() which returns the LiveData list of playlist, therefore, I added a comments that it fetches the playlist which is added by the user.

Reply:
If adding KDoc, go all the way and describe the function as KDoc recommends: https://kotlinlang.org/docs/reference/kotlin-doc.html